vyper/builtins/functions.py

```diff
@@ -294,6 +294,46 @@ class Slice(BuiltinFunction):
     ]
     _return_type = None
 
+    def evaluate(self, node):
+        validate_call_args(node, 3)
+        literal_value, start, length = node.args
+        if (
+            isinstance(literal_value, (vy_ast.Bytes, vy_ast.Str, vy_ast.Hex))
+            and isinstance(start, vy_ast.Int)
+            and isinstance(length, vy_ast.Int)
+        ):
+            (st_val, le_val) = (start.value, length.value)
```

Comment:
can these be spelled out? `start_val, length_val`

Reply:
Of course, thanks!